Command line remove crashes when no input is given

```
mono ckan.exe remove
About to remove:

 * 

Continue? [Y/N]

Unhandled Exception:
System.ArgumentNullException: Argument cannot be null.
Parameter name: key
  at System.Collections.Generic.Dictionary`2[System.String,CKAN.InstalledModule].ContainsKey (System.String key) [0x00000] in <filename unknown>:0 
  at CKAN.Registry.InstalledVersion (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.Registry.IsInstalled (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.Uninstall (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.UninstallList (IEnumerable`1 mods) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.UninstallList (System.String mod) [0x00000] in <filename unknown>:0 
  at CKAN.CmdLine.MainClass.Remove (CKAN.CmdLine.RemoveOptions options) [0x00000] in <filename unknown>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x00000] in <filename unknown>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.ArgumentNullException: Argument cannot be null.
Parameter name: key
  at System.Collections.Generic.Dictionary`2[System.String,CKAN.InstalledModule].ContainsKey (System.String key) [0x00000] in <filename unknown>:0 
  at CKAN.Registry.InstalledVersion (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.Registry.IsInstalled (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.Uninstall (System.String modName) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.UninstallList (IEnumerable`1 mods) [0x00000] in <filename unknown>:0 
  at CKAN.ModuleInstaller.UninstallList (System.String mod) [0x00000] in <filename unknown>:0 
  at CKAN.CmdLine.MainClass.Remove (CKAN.CmdLine.RemoveOptions options) [0x00000] in <filename unknown>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x00000] in <filename unknown>:0

```

--- a/CKAN/CmdLine/Main.cs
+++ b/CKAN/CmdLine/Main.cs
@@ -238,10 +238,17 @@ namespace CKAN.CmdLine
         // Uninstalls a module, if it exists.
         private static int Remove(RemoveOptions options)
         {
-            var installer = ModuleInstaller.Instance;
-            installer.UninstallList(options.Modname);
-
-            return Exit.OK;
+            if (options.Modname != null && options.Modname.Length > 0)
+            {
+                var installer = ModuleInstaller.Instance;
+                installer.UninstallList(options.Modname);
+                return Exit.OK;
+            }
+            else
+            {
+                User.WriteLine("No mod selected, nothing to do");
+                return Exit.BADOPT;
+            }
         }
 
         // TODO: This needs work! See GH #160.

